debian/kano-profile-i18n-orig.install

```diff
@@ -0,0 +1 @@
+po/messages.pot /mnt/translations/translations/kano-profile/
```

Comment:
Hmm, is there nothing we can do to prevent this hard-coded path? Probably not...